spec/openapi.yaml

```diff
@@ -1132,6 +1163,60 @@ components:
           description: Package version available for update
           example: "4.2-RC1"
 
+    Transfer:
+      type: object
+      properties:
+        type:
+          type: string
+          enum: [ NO_TRANSFER, FROM_WEB, FROM_CONNECT, FROM_PRINTER, FROM_SLICER, FROM_CLIENT, TO_CONNECT, TO_CLIENT ]
+          example: FROM_WEB
+        display_name:
+          type: string
+          description: Long Filename
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        path:
+          type: string
+          example: "/local"
+        display_path:
+          type: string
+          example: "/PrusaLink gcodes/"
+        url:
+          type: string
+          example: "https://files.printables.com/media/prints/42/gcodes/42_b42-242-442-8142c-424242/spice_harvester_0.3mm_pla_mk3s_12m.gcode"
```

Comment:
What kind of url is this? The example seems like a download from _external url_, which buddy doesn't support, is this also necessary for other types of downloads, seems weird to me, that it is  _required_. What is the purpose of this?